shared/django_apps/core/models.py

```diff
@@ -508,3 +508,10 @@ class CommitError(ExportModelOperationsMixin("core.commit_error"), BaseCodecovMo
 
     class Meta:
         app_label = CORE_APP_LABEL
+
+
+class UserOnboardingLifeCycleMetrics(models.Model):
```

Comment:
Thoughts on making this a standalone django app? Or ist here a strong reason this should be in `core`?

Reply:
I can do that